ui/templates/learningobjects/create_repo.html

```diff
@@ -0,0 +1,13 @@
+<h1>Create repository</h1>
```

Comment:
Yep, no problem. This is another thing I'll do after the listing page branch is merged into this.